answer_files/10_insider/Autounattend.xml

```diff
@@ -42,8 +42,8 @@
                     <InstallToAvailablePartition>false</InstallToAvailablePartition>
                     <InstallFrom>
                         <MetaData wcm:action="add">
-                            <Key>/IMAGE/NAME</Key>
-                            <Value>Windows 10 Enterprise</Value>
+                            <Key>/IMAGE/EditionID</Key>
```

Comment:
Nice. Where can I read more about /IMAGE/EditionID ?
How to list the edition(s) of a WIM file?
dism and `Get-WindowsImage` does not show me that. :-)

Reply:
I saw it in the Windows setup logs (c:\windows\panther\setuplog.act) actually . Shift-f10 brings you a cmd window while setup is running so if you can see the edition selection UI - that info should already be in setupact.log. IMAGE/INDEX also works according to https://social.technet.microsoft.com/Forums/windows/en-US/efdc699a-9604-4a87-9549-9d3e37a6312d/autounattendxml-stops-on-image-selection-section-in-the-setup?forum=w7itproinstall

The official way to produce unattend.xml is to use the Windows System Image Manager which can generate the XML. However, it's mostly just a UI skin on top of an XML schema. It can validate that the XML is valid, but not predict if setup will succeed.